config_sample.toml

```diff
@@ -0,0 +1,49 @@
+keystore_path = "/home/dilate/.ethereum/testnet/keystore"
```

Comment:
Please use the spelling of the CLI options (i.e. `-` instead of `_`). Converting to the internal name is simply `name.replace('-', '_')`.

Also, please comment out values that match the default. To differentiate between actual comments and defaults it's common to use a single `#` for defaults and double `##` for comments.